src/main/java/com/squareup/phrase/Phrase.java

```diff
@@ -182,6 +184,14 @@ public CharSequence format() {
     return formatted;
   }
 
+  /** Apply the phrase to the target. */
+  public void into(@NonNull TextView target){
```

Comment:
rename param name to `textView`? or `view`? Also update javadoc to reflect this.